src/components/icons/svg/external-link.react.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
All sizes should be `24px`. Please check with the design team

Reply:
you should just be able to change them yourself manually though

Reply:
Ok, I'll talk to Sven and ask him to send them again 😢

Reply:
I think you can just do it yourself, just modifiy the width and viewbox manually

Reply:
The icons have been replaced 👍 (exported again by sketch...)